outputs/core_pools.json

```diff
@@ -41,26 +42,35 @@
   },
   "arbitrum": {
     "0x125bc5a031b2db6733bfa35d914ffa428095978b000200000000000000000514": "ECLP-AUSDC-AUSDT",
+    "0x260dbd54d87a10a0fc9d08622ebc969a3bf4e6bb000200000000000000000536": "jitoSOL/wstETH",
     "0x2ce4457acac29da4736ae6f5cd9f583a6b335c270000000000000000000004dc": "sFRAX/4POOL",
     "0x2d6ced12420a9af5a83765a8c48be2afcd1a8feb000000000000000000000500": "cbETH/rETH/wstETH",
     "0x3fd4954a851ead144c2ff72b1f5a38ea5976bd54000000000000000000000480": "ankrETH/wstETH-BPT",
     "0x451b0afd69ace11ec0ac339033d54d2543b088a80000000000000000000004d5": "plsRDNT-Stable",
     "0x9791d590788598535278552eecd4b211bfc790cb000000000000000000000498": "wstETH-WETH-BPT",
     "0xb61371ab661b1acec81c699854d2f911070c059e000000000000000000000516": "ezETH/wstETH",
     "0xc2598280bfea1fe18dfcabd21c7165c40c6859d30000000000000000000004f3": "wstETH/sfrxETH",
+    "0xca8ecd05a289b1fbc2e0eaec07360c4bfec07b6100020000000000000000051d": "2CLP-AUSDC-USDC",
     "0xd0ec47c54ca5e20aaae4616c25c825c7f48d40690000000000000000000004ef": "rETH/wETH BPT",
+    "0xef0c116a2818a5b1a5d836a291856a321f43c2fb00020000000000000000053a": "ECLP-WOETH-WETH",
     "0x423a1323c871abc9d89eb06855bf5347048fc4a5000000000000000000000496": "4POOL",
     "0x32df62dc3aed2cd6224193052ce665dc181658410002000000000000000003bd": "RDNT-WETH",
     "0x9f8ed1acfe0c863381b9081aff2144fc867aa7730002000000000000000004d4": "ANKR:ankrETH"
   },
-  "optimism": {},
```

Comment:
We don't manage Op, this is correct, although it's also ok to have a list there.  It's not used.

Reply:
yea this is due to a change upstream, where core pools is only generated for balancer's production chains: https://github.com/BalancerMaxis/bal_addresses/pull/313/files